src/helpers/index.tsx

```diff
@@ -64,17 +64,15 @@ export async function getV1MarketPrice() {
  * @returns INTEGER usd value
  */
 export async function getTokenPrice(tokenId = "olympus") {
-  let resp;
-  try {
-    resp = await axios.get(`https://api.coingecko.com/api/v3/simple/price?ids=${tokenId}&vs_currencies=usd`);
-    return resp.data[tokenId].usd;
-  } catch (e) {
-    // console.log("coingecko api error: ", e);
-  }
+  const resp = (await axios.get(`https://api.coingecko.com/api/v3/simple/price?ids=${tokenId}&vs_currencies=usd`)) as {
+    data: { [id: string]: { usd: number } };
+  };
+  let tokenPrice: number = resp.data[tokenId].usd;
+  return tokenPrice;
```

Comment:
What's the background on this one? getting rid of the error handling?

Reply:
This did not come from me. It looks like someone else noticed the `unknown` static check error and implemented an alternative fix.

Reply:
the type should be fixed rather than removing the error handling.
The type should be:

```js
let resp: number
```

Reply:
I tried to suggest a fix here. A couple of comments:

1. `let resp: number` does not work , because `resp` is a composite type. `resp.data[tokenId].usd` is a number provided that axios returns the full data structure.
2. Exception handling is not ideal. Catching an error and returning nothing fails the TS static type checking. I put `return 0` in case of exception. However that does not inform the invoking function that there was an error. It will incorrectly think that the actual value of the token is 0. The business logic needs to be thought through.
3. Consider enforinc CI checks for build, test and lint (included in this PR) as well as pre-commit lint check (@alexleventer PR #1125 ) to ensure that this type of TS static code errors do not get into `develop` in the first place. 
4. Consider installing a [PR size labeling](https://github.com/marketplace/pull-request-size) app in github to help us keep PRs scoped down. 
5. Consider enforcing [small sized PRs as a best practice](https://www.netlify.com/blog/2020/03/31/how-to-scope-down-prs/ ).

Reply:
I hear what you're saying about 3, 4, & 5. And I think that's how this started. Why is this PR changing this code?
You said it didn't come from you. So would you be opposed to simply removing this change?